net-voip/mumble/mumble-1.5.517.recipe

```diff
@@ -0,0 +1,101 @@
+SUMMARY="Low latency encrypted VoIP client"
+DESCRIPTION="Mumble is a low-latency, high quality voice chat program primarily \
+intended for gaming. \
+Features include a encrypted connection with Public/private-key authentication and \
+noise suppression. \
+To use Mumble, you need a mumble server. You can either join a public existing server \
+or host your own."
+HOMEPAGE="https://www.mumble.info/"
+COPYRIGHT="2005-2022 The Mumble Developers"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/mumble-voip/mumble/releases/download/v1.5.517/mumble-1.5.517.tar.gz"
+CHECKSUM_SHA256="8efd06a0f47ebfe27735f7575ba0becfb21d0f400a5082202dbbd998ab8351de"
+PATCHES="mumble-1.5.517.patchset"
+ADDITIONAL_FILES="mumble.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	mumble$secondaryArchSuffix
+	app:Mumble
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libboost_system$secondaryArchSuffix >= 1.83.0
+	lib:libexpat$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	lib:libopus$secondaryArchSuffix
+	lib:libpcre2_posix$secondaryArchSuffix
+	poco$secondaryArchSuffix
+	lib:libportaudio$secondaryArchSuffix
+	lib:libprotobuf$secondaryArchSuffix
+	lib:libprotoc$secondaryArchSuffix
+	lib:libsndfile$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libspeexdsp$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:librnnoise$secondaryArchSuffix
+	lib:libX11$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libboost_system$secondaryArchSuffix >= 1.83.0
+	devel:libexpat$secondaryArchSuffix
+	devel:libogg$secondaryArchSuffix
+	devel:libopus$secondaryArchSuffix
+	devel:libpcre2_posix$secondaryArchSuffix
+	devel:libPocoFoundation$secondaryArchSuffix
+	devel:libprotobuf$secondaryArchSuffix
+	devel:libprotoc$secondaryArchSuffix
+	devel:libsndfile$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libspeexdsp$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:librnnoise$secondaryArchSuffix
+	devel:libX11$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cc
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:linguist$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:python3
+	"
+
+BUILD() {
+	 cmake -Bbuild -Dbundled-opus=OFF -Dbundled-rnnoise=OFF -Dbundled-speex=OFF \
+	 	-Dsymbols=ON -Dserver=OFF -Dclient=ON -Ddbus=OFF -Dzeroconf=OFF -Dtracy=OFF \
+	 	-DTRACY_ENABLE=OFF -Doverlay=OFF -Dtests=OFF -Dplugins=OFF \
+	 	-DCMAKE_BUILD_TYPE=RelWithDebInfo -Dwarnings-as-errors=OFF
```

Comment:
Could you sort this one line per option, this is heard to read now :)
Also (sugestion), use:
```
cmake -Bbuild -S.
...
make -Cbuild
```